ui-pf4/src/main/webapp/src/layout/TransformationPaths.ts

```diff
@@ -98,6 +98,14 @@ export const JAKARTA9: TransformationPathOption = {
   options: "jakarta-ee",
   iconSrc: migration,
 };
+export const JWS7: TransformationPathOption = {
```

Comment:
Is there a reason the JWS6 target constant is called JWS7?

Reply:
Thank you for noticing it @m-brophy ! It was a mistake of mine. I've just pushed a new commit renaming the variable

Reply:
Thanks @carlosthe19916  I'll do a test then approve and merge